autogpt_platform/frontend/src/components/Flow.tsx

```diff
@@ -433,6 +476,15 @@ const FlowEditor: React.FC<{
       setNodeId((prevId) => prevId + 1);
       clearNodesStatusAndOutput(); // Clear status and output when a new node is added
 
+      setViewport(
+        {
+          x: -viewportCoordinates.x * zoom + window.innerWidth / 2,
+          y: -viewportCoordinates.y * zoom + window.innerHeight / 2 - 100,
+          zoom: 0.8,
```

Comment:
Why is this 0.8 not 1.0 ?

Reply:
I have tried different values for zoom, and 0.8 looks visually better, but we could also use 1.0.